HeavyIonsAnalysis/JetAnalysis/python/inclusiveJetAnalyzer_cff.py

```diff
@@ -22,7 +22,8 @@
                                       doHiJetID = cms.untracked.bool(True),
                                       doStandardJetID = cms.untracked.bool(False),
                                       doSubEvent = cms.untracked.bool(False),
-                                      
+	                              jetIDWeight = cms.untracked.string("HeavyIonsAnalysis/JetAnalysis/data/JetID_TMVAClassification_BDTG_weights.xml"),       
```

Comment:
I wonder if your previous issues with the weights file not being picked up with crab etc are because HeavyIonsAnalysis/JetAnalysis is not an official CMSSW package, so the handling of the /data/ subfolder might be wonky.